system/hooks/rubocop_pre_commit.rb

```diff
@@ -3,12 +3,12 @@
 require 'english'
 require 'rubocop'
 
-ADDED_OR_MODIFIED = /A|AM|^M/.freeze
+ADDED_OR_MODIFIED = /A|AM|^M/
```

Comment:
Due to [Style/RedundantFreeze](https://docs.rubocop.org/rubocop/cops_style.html#styleredundantfreeze).